angular/projects/admin-nrpti/src/app/records/records-list/records-list.component.ts

```diff
@@ -172,6 +172,12 @@ export class RecordsListComponent implements OnInit, OnDestroy {
     });
   }
 
+  /**
+   * Clears the active keyword search text
+   */
+  clearSearchTerms() {
+    this.keywordSearchWords = '';
```

Comment:
Does the `X` button need to trigger the search when clicked?

Reply:
Good question. Not mentioned in the ticket, but kinda makes sense. 

Note that currently that clear advanced search does not re-search either.

Reply:
Oh gotcha.  If the current clear doesn't trigger the search, then seems fine that this wouldn't either. They should at least be consistent with eachother, I think.

Reply:
Oh gotcha.  If the current clear doesn't trigger the search, then seems fine that this wouldn't either. They should at least be consistent with eachother, I think.